data/grc/Arnott_1996_Athenaeus/stemma.gv

```diff
@@ -0,0 +1,23 @@
+# Athenaeus: Arnott 1996
+# Epitome derived from A, contamination from a sibling of A
+
+digraph {
+
+    omega -> A;
+    omega -> sibling_of_A;
+    A -> epitome;
+    sibling_of_A -> epitome [style="dashed"]; # hypothetical ms. derived from the archetype, by definition
```

Comment:
Could it be seen as extra-stemmatic contamination ? In which case, there would be a distinct archetype alpha, ancestor of A and not of his hypothetical ms.
If not, rename omega as alpha ?

Reply:
Is it "extra-stemmatic" if the question mark here stands for an hypothetical manuscript?

The statement to justify the stemma is this: "and so it may be wiser to believe that [A] was the main but not the only source of Epit[ome]".

I think the best way not to make assumptions about what the editor meant is to remove `omega -> sibling_of_A;`, so that the hypothetical manuscript (grey ellipse) floats in the air, while the contamination is hypothesised (black ellipse).

As for the name, "omega" or "alpha", see my other answer.

Reply:
Do you mean: no label for the archetype, so that we don't choose between "alpha" (used by this editor) and "omega" (conventional, and also used by the other editors)?

I just want to be sure that you don't see this as a reason not to remove `omega -> sibling_of_A`.

Once I review this thread of comments, I will update my PR. Things may be clearer then.